src/components/team-session/component.tsx

```diff
@@ -124,20 +168,36 @@ const TeamSession: React.FC<Props> = ({ team }) => {
               )}
           </header>
 
-          <ul className="flex-1 py-4 space-y-4 overflow-auto scrolling-touch">
+          <ul className="flex-1 py-4 space-y-4 scrolling-touch">
             {teamSession && onlineUsers
               .filter((item) => item.id !== teamSession.team_by_pk.entry?.user.id)
               .map((item) => (
-                <li key={item.id} className="flex items-center px-4 space-x-4">
-                  <img src={item.picture} alt={item.name} className="w-12 h-12 rounded-full" />
+                <li key={item.id} className="inline-flex flex-col px-4">
+                  <div className="relative">
+                    <div className="overflow-hidden">
+                      <img src={item.picture} alt={item.name} className="w-12 h-12 rounded-full" />
+                      {teamSession?.team_by_pk.entry?.reactions.filter(r => r.user_id === item.id).map((reaction) => {
+                        return (
+                          <>
```

Comment:
Are you really supposed to include the key twice like this or just make sure to wrap these in a parent div with a key-prop?